turtlebot3c-teleop-snap/snap/snapcraft.yaml

```diff
@@ -0,0 +1,75 @@
+name: turtlebot3c-teleop
+version: '0.1'
+license: GPL-3.0
+summary: Turtlebot3c teleop snap
+description: |
+  This snap automatically spawn a roscore and the core components for the
+  Turtlebot3.
+
+  This snap is meant to run on the Turtlebot3 robot or with the simulation
+
+  By default the snap runs on the robot.
+  To switch to the simulation
+
+    snap set turtlebot3c simulation=true
+
+  By default the snap used the lidar LDS-01 robot.
+  To select the other lidar for the robot
+
+    snap set turtlebot3c lds-model=LDS-02
+
+  By default the snap runs the waffle_pi Turtlebot3 model.
+  To select another robot model (waffle or burger)
+
+    snap set turtlebot3c turtlebot3-model=burger
+
+grade: stable
+confinement: strict
+base: core20 # Noetic
+
+architectures:
+  - build-on: amd64
+  - build-on: arm64
+  - build-on: armhf
+
+parts:
+  workspace:
+    plugin: catkin
+    source: https://github.com/ubuntu-robotics/turtlebot3c.git
+    source-branch: artivis-patch-1
+    source-subdir: turtlebot3c_teleop
+
+  # copy local scripts to the snap usr/bin
+  local-files:
+    no_extension: True
+    plugin: dump
+    source: snap/local/
+    organize:
+      '*.sh': usr/bin/
+
+apps:
+  core:
+    # daemon: simple
```

Comment:
Why not a daemon?

Reply:
It will be, I simply didn't test it yet.